vnext/Microsoft.ReactNative/ReactInstanceSettings.h

```diff
@@ -11,12 +11,14 @@
 #define REACT_DEFAULT_USE_FAST_REFRESH true
 #define REACT_DEFAULT_USE_LIVE_RELOAD false
 #define REACT_DEFAULT_ENABLE_DEVELOPER_MENU true
+#define REACT_DEFAULT_DEBUGGER_PORT 9229
 #else
 #define REACT_DEFAULT_USE_DEVELOPER_SUPPORT false
 #define REACT_DEFAULT_USE_WEB_DEBUGGER false
 #define REACT_DEFAULT_USE_FAST_REFRESH false
 #define REACT_DEFAULT_USE_LIVE_RELOAD false
 #define REACT_DEFAULT_ENABLE_DEVELOPER_MENU false
+#define REACT_DEFAULT_DEBUGGER_PORT 0
```

Comment:
It's possible a developer will want to do direct debugging on release builds, I suggest always setting the default to 9229